src/main/java/DialogGenerator.java

```diff
@@ -0,0 +1,54 @@
+public class DialogGenerator {
+    public void printLine(){
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public void greeting(){
```

Comment:
should require a space between the opening brace, ie.

`public void greeting () {`